addons/Makefile.rel.am

```diff
@@ -0,0 +1,17 @@
+include %D%/src/Makefile.rel.am
+
+if ENABLE_ADDONS
+addonsdocdir = $(docdir)/addons
+dist_addonsdoc_DATA = \
+	%D%/README \
+	%D%/ChangeLog \
+	%D%/NEWS \
+	%D%/COPYING \
+	%D%/AUTHORS
+
+EXTRA_DIST += \
+	%D%/wscript_build \
+	%D%/wscript_configure
+
+plugin = addons
```

Comment:
`plugin=` will break stuff if it's used, as it will keep getting redefined when using the toplevel Makefile.
Though, it seem to just be a leftover and not be used anymore, is it?